packages/components/src/components/radio-group/radio-group.scss

```diff
@@ -1,14 +1,8 @@
 .dc-radio-group {
     display: flex;
     margin-top: 16px;
-    &--is-center {
-        flex-direction: row;
-        align-items: center;
-    }
-    &--is-left {
-        flex-direction: column;
-        align-items: flex-start;
-    }
+    flex-direction: row;
+    align-items: center;
```

Comment:
Are you sure it is not affecting all other RadioGroups?

Reply:
by default this is how it is. I added `is_left` as a prop to apply the stylings to make it vertical rather than horizontal.